nav2_bringup/launch/nav2_params.yaml

```diff
@@ -22,21 +22,23 @@ local_costmap:
   local_costmap:
     ros__parameters:
       obstacle_layer:
-        enabled: False
+        enabled: True
```

Comment:
I'd prefer to leave the obstacle layer off by default unless you've run the system test enough times to ensure the success rate hasn't gone down by doing this.

Reply:
OK, I've changed it